pkgs/development/compilers/llvm/11/lld/default.nix

```diff
@@ -12,15 +13,19 @@ stdenv.mkDerivation rec {
 
   src = fetch pname "1kk61i7z5bi9i11rzsd2b388d42if1c7a45zkaa4mk0yps67hyh1";
 
+  patches = [
+    ./gnu-install-dirs.patch
+  ];
+
   nativeBuildInputs = [ cmake ];
-  buildInputs = [ llvm libxml2 ];
+  buildInputs = [ llvm.dev libxml2 ];
 
-  outputs = [ "out" "dev" ];
+  cmakeFlags = lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
+    "-DLLVM_TABLEGEN_EXE=${buildPackages.llvm_10}/bin/llvm-tblgen"
```

Comment:
Trying to cross compile lld fails with 

```
CMake Error at CMakeLists.txt:42 (message):
  LLVMConfig.cmake not found
```

It looks like the directory emitted by `llvm-config-native --cmakelists` is `${llvm.dev}/lib64/cmake/llvm`, but the cmake files are contained in `${llvm.dev}/lib/cmake/llvm`

(Also, this is referencing `llvm_10` as `llvm/11/lld`).

Reply:
Mmm, we should set `LLVM_LIBDIR_SUFFIX` and friends to be the empty string for that.

I wonder also if anything is looking up `ClangConfig.cmake`, because we could have problems with that being expected to be in `$out`.